aiter/ops/triton/moe_op.py

```diff
@@ -17,7 +17,7 @@
 
 _MOE_A_QUANT_FUNC = dynamic_per_tensor_quant_fp8_i8
 
-_USE_MOE_PERSISTENT_KERNEL = False
+_USE_MOE_PERSISTENT_KERNEL = True
```

Comment:
Do we want this to be default?

Reply:
Still a draft pr atm, we are testing things with this :)